### PR TITLE
 Fixes#251 Updates MozillaAllHands16 sample

### PR DESCRIPTION
**Pull request by ParthS007**
Updates MozillaAllHands16 sample
Live Link: http://opev-webgen-dev.herokuapp.com/live/preview/parth1989shandilya@gmail.com/AllHands/2016Hawaii/

@mariobehling @geekyd Please review

**I will add it to overview site after merging of this PR**

### CODE REVIEW

**Comment by geekyd** [comment]
@ParthS007 The provided link is not working. Can you update the link?

**Comment by ParthS007** [comment]
@geekyd  Please wait for 2 min, updating it again in all 3 PRs

**Comment by ParthS007** [comment]
@geekyd  Done !

File: sample/MozillaAllHands16/event [hunk 1]
```diff
@@ -1,90 +1,90 @@
...
+					"link": "https://twitter.com/mozilla",
+					"name": "Twitter"
 		},
 		{
-			"id": 2,
-			"link": "https://github.com/mozilla/mofo-allhands/tree/gh-pages/2016-hawaii",
-			"name": "Github"
+					"id": 2,
+					"link": "https://github.com/mozilla/mofo-allhands/tree/gh-pages/2016-hawaii",
+					"name": "Github"
```
Comment (agbilotia1998) [inline sample/MozillaAllHands16/event]: @ParthS007 This indentation is incorrect.

Comment (ParthS007) [reply sample/MozillaAllHands16/event]: Thanks @agbilotia1998 , will do the changes

**Comment by ParthS007** [comment]
@mariobehling @SaptakS @geekyd @agbilotia1998 Please review.

**Pull request merged**